docs/pages/eas/environment-variables/without-eas.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Using environment variables without EAS
+sidebar_title: Without EAS
+description: Learn about non-EAS ways to manage environment variables in Expo and React Native projects.
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+[EAS](/eas/environment-variables/) is the recommended way to manage environment variables for cloud builds and updates, but you can still work locally or with other tooling.
+
+## Managing environment variables without EAS
+
+If you want to manage environment variables without EAS, you can use tools like [`dotenv`](https://www.npmjs.com/package/dotenv) (Node-based loaders) or services such as [Doppler](https://www.doppler.com/) that inject environment variables. These utilities allow you to create a **.env** file in which you can store your environment variables.
+
+> **Note:** Avoid committing secrets to **.env** files if you are managing your environment variables without EAS.
+
+## How environment variables are loaded
+
+After creating **.env** file, you need to ensure that the file is committed to your project's Version Control System and is not listed inside **.gitignore** file. A **.env** file committed to your project, is picked by EAS commands like `eas build`, `eas update`, and so on.
+
+The **.env** files load according to the standard .env file resolution and then replaces all references in your code to `process.env.EXPO_PUBLIC_[VARIABLE_NAME]` with the corresponding value set in the .env files. Code inside **node_modules** directory is not affected for security purposes.
```

Comment:
Let's link to https://github.com/bkeepers/dotenv/blob/c6e583a/README.md#what-other-env-files-can-i-use for "standard .env file resolution"